resources/views/notification/simple-html.blade.php

```diff
@@ -0,0 +1,5 @@
+@extends('notification.base-html')
+@section('content')
+    <h1>Just a quick note..</h1>
```

Comment:
This "simple" template is for notifications that are simple enough to only need a single sentence/line. We could drop the title altogether, but this is just a straight port of the `email.notification` template and it had this structure.